verification/curator-service/ui/cypress/integration/components/EditCase.spec.ts

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-undef */
+describe('Edit case', function () {
+    beforeEach(() => {
+        cy.task('clearCasesDB', {});
+        cy.login();
+    });
+
+    afterEach(() => {
+        cy.clearSeededLocations();
+    });
+
+    it('Errors when case does not exist', function () {
+        cy.visit('/cases/edit/foo');
+        cy.contains('Request failed');
+    });
+
+    it('can edit an existing case', function () {
+        cy.addCase({
+            country: 'France',
+            notes: 'some notes',
+            sourceUrl: 'www.example.com',
+        });
+        cy.request({ method: 'GET', url: '/api/cases' }).then((resp) => {
+            console.log(resp.body);
+            expect(resp.body.cases).to.have.lengthOf(1);
+            cy.visit(`/cases/edit/${resp.body.cases[0]._id}`);
+            // Check that we have something from the original case.
+            cy.contains('France');
```

Comment:
Could also check 'Female' and '21' don't exist, to ensure the edit is changing something

Reply:
good call, done.